src/test/java/seedu/address/model/person/PersonTest.java

```diff
@@ -35,16 +35,24 @@ public void isSamePerson() {
         // null -> returns false
         assertFalse(ALICE.isSamePerson(null));
 
-        // same name, all other attributes different -> returns true
-        Person editedAlice = new PersonBuilder(ALICE).withPhone(VALID_PHONE_BOB).withEmail(VALID_EMAIL_BOB)
-                .withAddress(VALID_ADDRESS_BOB).withSchedule(VALID_SCHEDULE_BOB).build();
+        // same name and phone, all other attributes different -> returns true
+        Person editedAlice = new PersonBuilder(ALICE).withEmail(VALID_EMAIL_BOB)
+                                                     .withAddress(VALID_ADDRESS_BOB).withSchedule(VALID_SCHEDULE_BOB)
+                                                     .withSubject(VALID_SUBJECT_BOB).withRate(VALID_RATE_BOB).build();
```

Comment:
I think there is too much indentation for these 2 lines.

Reply:
Okay, fixed